tutorials/3d/inverse_kinematics.rst

```diff
@@ -17,18 +17,18 @@ move each joint in our arm to get to that target.
 Initial problem
 ~~~~~~~~~~~~~~~
 
-Talking in Godot terminology, the task we want to solve here is to position
+In Godot parlance, the task we want to solve here is to position
```

Comment:
We should remove the "In Godot parlance", it doesn't make sense in context and "parlance" is not a very accessible word for many people. How about instead we just use: "We want to position the 2 angles..."

Reply:
> We should remove the "In Godot parlance", [...] "parlance" is not a very accessible word for many people. How about instead we just use: "We want to position the 2 angles..."

@clayjohn : Amended.

> "In Godot parlance", [...] doesn't make sense in context

I had rephrased the awkward wording intended to point out that that was not an everyday way of looking at such a task, but a specialized, technical, algorithmic manner of breaking down a problem into steps and procedures that can be carried out by a computer agent.